examples/mean/tests/Makefile

```diff
@@ -1,33 +1,21 @@
-
 TOPLEVEL_LANG ?= verilog
 
-ifneq ($(TOPLEVEL_LANG),verilog)
-
-all:
-	@echo "Skipping example due to TOPLEVEL_LANG=$(TOPLEVEL_LANG) not being verilog"
-clean::
-
-else
-
-TOPLEVEL := mean_sv
-
 PWD=$(shell pwd)
 
-VHDL_SOURCES = $(PWD)/../hdl/mean_pkg.vhd
-VHDL_SOURCES += $(PWD)/../hdl/mean.vhd
-
-ifeq ($(SIM),questa)
-COMPILE_ARGS = -mixedsvvh
+ifeq ($(TOPLEVEL_LANG),verilog)
+    VERILOG_SOURCES = $(PWD)/../hdl/mean.sv
```

Comment:
I'd use ``+=`` here too for consistency.

Reply:
OK